bc_obps/service/user_operator_service.py

```diff
@@ -164,8 +156,22 @@ def list_user_operators(page: int = 1, sort_field: str = "created_at", sort_orde
         )
 
     @transaction.atomic()
-    def create_operator_and_user_operator(updated_data: UserOperatorOperatorIn, user_guid: UUID):
-        "Function to create a user_operator and an operator (new operator that doesn't exist yet)"
+    def create_operator_and_user_operator(updated_data: UserOperatorOperatorIn, user_guid: UUID) -> Dict:
+        """
+        Function to create a user_operator and an operator (new operator that doesn't exist yet).
+
+        Parameters:
+            updated_data: Updated data for the user operator.
```

Comment:
Since this function is about creation, maybe the arg should be `new_data` instead of `updated_data`? Updated implies editing

Reply:
Valid point 👍